org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/command/HelpOnIconsCommand.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SonarLint for Eclipse
+ * Copyright (C) 2015-2023 SonarSource SA
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.eclipse.ui.internal.command;
+
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.sonarlint.eclipse.ui.internal.util.BrowserUtils;
+
+/** When the user wants to know what the issue / rule icons mean (and why they differ between old / new CCT) */
+public class HelpOnIconsCommand extends AbstractIssueCommand {
+  @Override
+  protected void execute(IMarker selectedMarker, IWorkbenchWindow window) {
+    // TODO: Change when documentation is ready
```

Comment:
Kevin is aware and I'll await the final documentation containing information on the icons